services/src/main/java/org/keycloak/theme/DefaultThemeManager.java

```diff
@@ -106,20 +115,19 @@ private Theme loadTheme(String name, Theme.Type type) {
         List<Theme> themes = new LinkedList<>();
         themes.add(theme);
 
-        if (theme.getImportName() != null) {
-            String[] s = theme.getImportName().split("/");
-            themes.add(findTheme(s[1], Theme.Type.valueOf(s[0].toUpperCase())));
-        }
+        if (!processImportedTheme(themes, theme, name, type)) return null;
 
         if (theme.getParentName() != null) {
             for (String parentName = theme.getParentName(); parentName != null; parentName = theme.getParentName()) {
+                String currentThemeName = theme.getName();
                 theme = findTheme(parentName, type);
+                if (theme == null) {
```

Comment:
Shouldn't this return the default theme?

Reply:
See my comment https://github.com/keycloak/keycloak/pull/17350#discussion_r1121871082